packages/terra-tag/tests/jest/__snapshots__/Tag.test.jsx.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Tag should render a tag 1`] = `
```

Comment:
all these `1`'s seam to be coming from your `it(...)` block descriptions. can we change the RHS of the arrow-function to return just an empty object?